pkg/query/api/v1.go

```diff
@@ -649,3 +661,20 @@ func (api *API) labelNames(r *http.Request) (interface{}, []error, *ApiError) {
 
 	return names, warnings, nil
 }
+
+// toThanosError converts a Prometheus execute error into one of the Thanos error constants and returns a generic
+// `errorExec` for unknown types.
+func toThanosError(e error) ErrorType {
```

Comment:
let's be explicit maybe?
```suggestion
func promqlToThanosErr(e error) ErrorType {
```